packages/eslint-plugin-query/src/utils/detect-react-query-imports.ts

```diff
@@ -55,7 +55,8 @@ export function detectTanstackQueryImports(create: EnhancedCreate): Create {
       ImportDeclaration(node) {
         if (
           node.specifiers.length > 0 &&
-          node.importKind === 'value' &&
+          // Change condition to check for undefined or 'value'
+          (node.importKind === 'value' || node.importKind === undefined) &&
```

Comment:
the comment is pretty unnecessary:

```suggestion
          (node.importKind === 'value' || node.importKind === undefined) &&
```